covid-vaccine-faq-bot/assets/make-your-own/studioflow-agent-generator.ipynb

```diff
@@ -0,0 +1,1687 @@
+
```

Comment:
This doesn't look right to me...I thought this was supposed to be a Python script. I tried to import this into a Jupiter notebook but it's not working. Are we sure this is the right file?

Reply:
@jme783 is this comment resolved?